### PR TITLE
Azure container apps - Email processor

### PR DESCRIPTION
**Pull request by nickdala**
* Email processor container app processing messages from service bus using spring cloud stream
* Terraform for Azure Container Apps and Azure Container Registry
* AZD deployment

### CODE REVIEW

File: apps/email-processor/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/openjdk/jdk:17-ubuntu
+VOLUME /tmp
```
Comment (mnriem) [inline apps/email-processor/Dockerfile:2]: Why is this volume needed?

File: infra/dev/terraform/aca.tf [hunk 2]
```diff
@@ -22,7 +43,11 @@ resource "azurerm_container_app" "container_app" {
   template {
     container {
       name   = "email-processor-app"
-      image  = "nickdala/cams-email-processor:0.1"
+
+      // A container image is required to deploy the ACA resource.
+      // Since the rendering service image is not available yet,
+      // we use a placeholder image for now.
+      image  = "mcr.microsoft.com/cbl-mariner/busybox:2.0"
```
Comment (mnriem) [inline infra/dev/terraform/aca.tf:50]: Is an actual functional image here required? If not, just use 'scratch'.

**Review by mnriem — APPROVED** [review]
I added two questions. Not blockers, but something to consider.

File: apps/email-processor/Dockerfile (hunk 1 above)
Comment (nickdala) [reply apps/email-processor/Dockerfile:2]: Removing the volume.  Not needed.

File: infra/dev/terraform/aca.tf (hunk 2 above)
Comment (nickdala) [reply infra/dev/terraform/aca.tf:50]: Good point.

**Pull request merged**